CHANGELOG.unreleased.md

```diff
@@ -7,6 +7,8 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
+- [Metadata Backup] The user must specify the target pool UUID for the restoration
```

Comment:
The changelog is for users, therefore only visible features should appear.

Keep this entry for the UI PR :slightly_smiling_face: